src/index.html

```diff
@@ -1,43 +1,50 @@
 <!DOCTYPE html>
 <html lang="en">
 
-  <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/fontawesome.min.css" integrity="sha384-jLKHWM3JRmfMU0A5x5AkjWkw/EYfGUAGagvnfryNV3F9VqM98XiIH7VBGVoxVSc7" crossorigin="anonymous">
-
-    <title>Spicy Foods</title>
-  </head>
-
-  <body>
-
-    <div class="modal-container" id="modal-container"></div>
-
-    <header>
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+	integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+	crossorigin="anonymous" referrerpolicy="no-referrer" />
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;600;700&family=Roboto&display=swap"
+		rel="stylesheet">
+	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+	<script src="./index.js" type="module"></script>
+  <title>Spicy Foods</title>
+</head>
+
+<body>
+
+  <div class="modal-container" id="modal-container"></div>
+
+  <header>
+    <div class="logo">
       <a href="#">
         <img class="logo" src="" alt="logo">
       </a>
+    </div>
```

Comment:
- [ Optional] Please consider fixing this image, it appears to be broken when the page loads. I think this is because the image has no value for the `src` attribute. Please see the screenshot below and rectify this.

## Screenshot
![Screenshot from 2023-04-28 23-01-45](https://user-images.githubusercontent.com/16392046/235243434-e47006df-1eaf-4fcf-9a5c-e35d14fb1cb7.png)